ts/hooks/useStartSupportRequest.ts

```diff
@@ -29,7 +29,7 @@ export const useStartSupportRequest = ({
    *  We have to use the deprecated currentRouteSelector because, at the moment, some components are rendered outside the navigation context.
```

Comment:
We can safely remove this comment since we're removing the legacy hook

Reply:
You're right, removed